docs/manual/tags.md

```diff
@@ -285,6 +285,10 @@ Sigsize           | 257  | int32        | Header+payload size.
 
 ## Installed packages
 
+Some information on how exactly a package got installed can not be
+known before hand. RPM though adds a few tags to the header during
+installation before adding it to the rpmdb.
```

Comment:
Some grammar/spelling issues in this paragraph. I'd go with just "The following tags do not exist in package files, they are only ever added during installation." or something like that.

Reply:
Does RPM correctly reject such tags in package files?

Reply:
Redone.